package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "trainee-ui-app",
-  "version": "0.94.0",
+  "version": "0.94.1",
```

Comment:
Just a tiny question as I have no idea at all. Do we bump up version for test fix?

Reply:
Good point. I'll actually revert that.